test/bbb_eth_listener_test/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Set a variable for commands below
```

Comment:
I'm gonna try and move both the bbb_eth_listener & nuc_eth_listener test under the same directory